op-mode-definitions/reset-ip-bgp.xml.in

```diff
@@ -143,6 +143,31 @@
                   </node>
                 </children>
               </tagNode>
+              <tagNode name="vrf">
+                <properties>
+                  <help>Clear BGP statistics or status for vrf</help>
+                  <completionHelp>
+                    <path>vrf name</path>
+                  </completionHelp>
+                </properties>
+                <children>
+                  <leafNode name="all">
+                    <properties>
+                      <help>Clear all BGP peering sessions for vrf</help>
+                    </properties>
+                    <command>vtysh -c "clear bgp vrf $5 *"</command>
+                  </leafNode>
+                  <leafNode name="node.tag">
+                    <properties>
+                      <help>Clear BGP neighbor IP address</help>
+                      <completionHelp>
+                        <list>&lt;x.x.x.x&gt;</list>
```

Comment:
What about IPv6 neighbors?

I think it would make sense to combine the ip and ipv6 bgp reset commands into `reset bgp <neighbor>` and `reset bgp vrf <vrf> <neighbor>`.

Reply:
Sometimes I wonder if we can get rid of separate commands for IPv4 and IPv6 neighbors that mainly exist because FRR has different commands for them. For a complete session reset, the session transport protocol is unimportant after all.

Reply:
@dmbaturin Sounds like we're on the same page

Reply:
Actually that is what FRR does by dropping the ip/ipv6 address family and implementing the e.g. `clear bgp` or `show bgp` tree. It supports passing any address family.

In our case this PR is good, it only lacks an additional entry to `<list>`. Will submit it in a subsequent commit. Thanks @sever-sever